src/Extensions/Abstractions/MachineInformation/EmptyMachineInformation.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Reflection;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Base class for machine information</summary>
+	public class EmptyMachineInformation : IMachineInformation
```

Comment:
DefaultMachineInformation would be a better name? As we have MachineCount = 1
Also is this supposed to be instantiated?

Reply:
I think it might create confusion with `BasicMachineInformation`, but we would need to change this interface in future PR so we can discuss renaming